core/src/components/picker-column/picker-column.tsx

```diff
@@ -33,6 +33,12 @@ export class PickerColumnCmp implements ComponentInterface {
   private rafId?: ReturnType<typeof requestAnimationFrame>;
   private tmrId?: ReturnType<typeof setTimeout>;
   private noAnimate = true;
+  // `colDidChange` is a flag that gets set when the column is changed
+  // dynamically. When this flag is set, the column will refresh
+  // to incorporate the new column data.
```

Comment:
```suggestion
  // after the component re-renders to incorporate the new column data.
  // This is necessary because `this.refresh` queries for the option elements,
  // so it needs to wait for the latest elements to be available in the DOM.
```